.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+
+    # Iterate and analyze each directory
+    for directory in directories:
+        for root, _, files in os.walk(directory, topdown=False):
+            for name in files:
+                # Read each file and count the lines found
+                result.append(os.path.join(root, name))
+    # Return
+    return result
+
+
+def _arg_parser_resolver():
```

Ruff (0.12.2)

Missing return type annotation for private function `_arg_parser_resolver`

(ANN202)